packages/es-components/src/components/containers/banner/Banner.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const UnstyledBanner = styled.div``;
+const Banner = styled(({ ...props }) => (
+  <UnstyledBanner {...props}></UnstyledBanner>
+))`
+  background-color: ${props => props.bgColor};
+  color: ${props => props.textColor};
+  align-items: center;
+  border-radius: 2px;
+  display: flex;
+  margin-bottom: 25px;
+  padding: 15px;
```

Comment:
Should the padding come from `theme`?

Reply:
Should the padding change based on media query?

Reply:
I'll take a look into it